src/deploy.js

```diff
@@ -27,14 +30,15 @@ const trackProgress = function(progressData) {
   };
 };
 
-module.exports = function(progressData, context, client, storage, config, slackTemplate) {
+module.exports = function(progressData, context, client, storage, config, slackTemplate, managementClientId) {
```

Comment:
Good point, I started with passing the name, then changed it and didn't notice config was passed in.